mh-journal-historian.js

```diff
@@ -64,13 +64,11 @@
 				console.log(mutation.target);
 			}
 		}
-		saveEntries();
-		renderBtns();
 
-		observer.observe(observerTarget, {
-			childList: true,
-			subtree: true
-		});
+		// Only save if something was added.
```

Comment:
is this portion critical to the PR or just some optimization? I tried this for a bit before dropping the need to check for added nodes. Maybe partly because of my lack of confidence in the mutation and xhr observers.

IIRC I left the duplicate checking to the saveEntries function because it didn't seem like a mutation would always be observed upon loading the page initially, and that's a time your journal may have a bunch of new entries from overnight or mobile, etc.

Also excuse my githubbing if this isn't the proper way to go about things, open to any guidance